resources/elasticbeanstalk_configuration_options.go

```diff
@@ -0,0 +1,163 @@
+package resources
+
+import (
+	"context"
+	"time"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk"
+	"github.com/aws/aws-sdk-go-v2/service/elasticbeanstalk/types"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func ElasticbeanstalkConfigurationOptions() *schema.Table {
```

Comment:
this should be a relation table to environments

Reply:
Done